sdk/lib/udmihelp/udmihelp.h

```diff
@@ -37,4 +36,3 @@ FreeSMBiosData(
 } // extern "C"
 #endif
 
-#endif /* UDMIHELP_H */
```

Comment:
You can remove remaining empty line from the EOF.

Reply:
done.